core/embed/rust/src/ui/display/toif.rs

```diff
@@ -271,6 +271,14 @@ impl<'i> Toif<'i> {
         Offset::new(self.width(), self.height())
     }
 
+    pub fn stride(&self) -> usize {
+        if self.is_grayscale() {
+            self.width() as usize / 2
```

Comment:
Not sure how this is intended to be used. The division for grayscale toif rounds down the number of pixels to even number, which might or might not be ok depending on context

Reply:
It's a bug, fixed here fixed 2536ed1b7195001706689a843df0cfd43b08340d